modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/search/spi/model/index/contributor/CommerceOrderModelDocumentContributor.java

```diff
@@ -247,6 +250,18 @@ private BigDecimal _getItemsQuantity(CommerceOrder commerceOrder) {
 		return count;
 	}
 
+	private double _getTotalAmount(CommerceOrder commerceOrder)
+		throws PortalException {
+
+		CommerceMoney commerceOrderTotalCommerceMoney =
+			commerceOrder.getTotalMoney();
+
+		BigDecimal commerceOrderTotalValue =
+			commerceOrderTotalCommerceMoney.getPrice();
+
+		return commerceOrderTotalValue.doubleValue();
+	}
+
```

Comment:
How we are indexing is the exact same as the way we are providing the value in the [DTOConverter](https://github.com/liferay-commerce/liferay-portal/blob/8b0ac72cc4e071ae3db73d07b6235f649b1193e5/modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/dto/v1_0/converter/OrderDTOConverter.java#L631) so we keep consistency in how users may input the values as well.